README.md

```diff
@@ -2,6 +2,43 @@
 
 Automate onboarding with https://github.com/packit/dist-git-to-source-git
 
+## Decommissioning
+
+**As of Q1 2022, we are planning to decommission the `c8` branches and repositories.**
+
+This should happen during 3 stages.
+
+### Stage 1
+
+1. Identify c8-only repos (see [packages.json](packages.json))
+
+2. Stop the d2s update service
```

Comment:
And archive `packit/dist-git-to-source-git` one. Clean up Jira, credentials in the credential store, delete (or ask for deletion) of the bot accounts in the services that are not used anymore (in git.centos.org for example).

Check if `packit/packit-service-centosmsg` is still needed, if not, that can be archived, too.

Maybe mark related research in the research repo as deprecated, to avoid future confusion.

Delete scraping the the `/metrics` endpoint from the internal Prometheus configuration and delete the Graphana dashboard.

Delete the corresponding maintain task from Jira :)

Reply:
uhhhhh...

not 3 or 5 story points any more :<

thanks for writing down such a detailed list